services/order.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Order, Ticket
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from typing import List, Dict
+
+User = get_user_model()
+
+
+@transaction.atomic
+def create_order(
+        tickets: List[Dict],
+        username: str,
+        date: str = None

```

Comment:
Consider using `Optional[str]` from the `typing` module for the `date` parameter to explicitly indicate that it can be `None`.